config/power_virtual_servers_regions.yml

```
@@ -0,0 +1,40 @@
+us-south:
+  :name: us-south
+  :description: Dallas, TX United States (deprecated)
+  :hostname: us-south.power-iaas.cloud.ibm.com
+eu-de:
+  :name: eu-de
+  :description: Frankfurt, Germany
+  :hostname: eu-de.power-iaas.cloud.ibm.com
+lon:
+  :name: London
```

We should be consistent and set `:name:` to the exact region sub-domain value:
```suggestion
  :name: lon
```

Same comment for the rest of the entries.